src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -551,13 +552,29 @@ def __call__(self, rcvd_messages: List[Tuple[int, Message]]):
             orders_data.add_orders(best_orders, overwrite=True)
             self.orders = orders_data
 
-            # filter out aggressive orders to allies
-            yield self.replace_aggressive_order_to_allies()
+
+            # fmt: off
+
+            self.allies = [pow for pow in powers if (pow != self.power_name and powers[pow] >= self.ally_threshold)]
+            self.foes = [pow for pow in powers if (pow != self.power_name and powers[pow] <= self.enemy_threshold)]
+            self.neutral = [pow for pow in powers if (pow != self.power_name and powers[pow] > self.enemy_threshold and powers[pow] < self.ally_threshold)]
+
+            # GLOBAL message and filter aggressive moves to allies are disabled in S1901M
+            if self.game.get_current_phase()!='S1901M':
```

Comment:
Umm I think we are supposed to send stance messages to GLOBAL but in first phase, just claim all are NEUTRAL

Reply:
oh I saw jbg said "i.e. don't print anything"